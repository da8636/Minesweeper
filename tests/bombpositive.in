g 10 10
b 0 0
b 10 1
