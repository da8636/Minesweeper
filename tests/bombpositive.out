g 10 10
b 0 0
error
